ROADMAP.md

```diff
@@ -74,12 +74,13 @@ Using `gfx-rs` for backend abstraction allows us to provide compatibility with m
 **Next steps**
  * Raw input support (winit?)
  * Gamepad support (winit?)
- * Input Mapping
+ * Input Mapping: Map input events (e.g key press) to abstract action events (`Jump`, `Forward`, ..)
```

Comment:
I don't really like using strings. It would be better to use types instead, so everything can be typechecked and more errors can be cought at compile time.